src/autorest-core/lib/configuration.ts

```diff
@@ -37,6 +37,8 @@ export interface AutoRestConfigurationImpl {
   "fluent"?: boolean;
   "azure-arm"?: boolean;
```

Comment:
`to determine whether to run the validator?` - only for legacy behavior, we already have more meaningful flags (https://github.com/Azure/autorest/blob/master/src/autorest-core/lib/pipeline/pipeline.ts#L110) and can innovate in that space.

What the flag really is for now is switching to Azure-style code gen - that's why we still have it.